src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -138,26 +139,4 @@ public class GeneratedTransaction : Transaction { }
     /// System transaction that is to be executed by the node without including in the block. 
     /// </summary>
     public class SystemTransaction : Transaction { }
-
-    public static class TransactionExtensions
```

Comment:
Move CalculateTransactionPotentialCost and CalculateEffectiveGasPrice to TransactionExtensions?